client-app/shared/wishlists/components/wishlist-line-items.vue

```diff
@@ -1,7 +1,22 @@
 <template>
   <VcLineItems :items="items" removable disable-subtotal @remove:items="$emit('remove:items', $event)">
+    <template #header-product>
+      <div class="text-center">
+        {{ $t("common.labels.product") }}
+      </div>
+    </template>
+    <template #header-properties>
+      <div class="text-center">
+        {{ $t("common.labels.properties") }}
+      </div>
+    </template>
+    <template #header-price>
+      <div class="text-center">
+        {{ $t("common.labels.price_per_item") }}
+      </div>
+    </template>
```

Comment:
I appreciate your wish to refactor this, but I will propose to do it in other task.
You make this header columns slots, but there is no same slots for columns in line item itself. Also, I think it's better to use another approach and put everything in default slot, because now you can't remove or add additional columns between these.

Reply:
Resolved as "leave as is" after discussion